x/fundraising/keeper/auction.go

```diff
@@ -303,7 +304,8 @@ func (k Keeper) UpdateAllowedBidder(ctx sdk.Context, auctionId uint64, bidder sd
 	return nil
 }
 
-// AllocateSellingCoin releases designated selling coin from the selling reserve account.
+// AllocateSellingCoin allocates allocated selling coin for all matched bids in MatchingInfo and
+// releases them from the selling reserve account.
 func (k Keeper) AllocateSellingCoin(ctx sdk.Context, auction types.AuctionI, mInfo MatchingInfo) error {
 	// Call the before seling coin distributed hook
 	k.BeforeSellingCoinsDistributed(ctx, auction.GetId(), mInfo.AllocationMap, mInfo.RefundMap)
```

Comment:
`BeforeSellingCoinsDistributed` can be replaced by `BeforeSellingCoinsAllocated` to be consistent with the functions.